uber-package-builder.sh

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env bash
+set -e
+
```

Comment:
(nit) a comment here might be helpful for those without project context. (e.g. what is this about and why we are doing this?)

Reply:
And someone will definitely ask that why "uber" :D